rest/taskana-rest-spring/src/test/java/pro/taskana/rest/TaskanaEngineControllerIntTest.java

```diff
@@ -49,9 +47,9 @@ void testClassificationTypes() {
             HttpMethod.GET,
             restHelper.defaultRequest(),
             ParameterizedTypeReference.forType(List.class));
-    assertTrue(response.getBody().contains("TASK"));
-    assertTrue(response.getBody().contains("DOCUMENT"));
-    assertFalse(response.getBody().contains("UNKNOWN"));
+    assertThat(response.getBody().contains("TASK")).isTrue();
```

Comment:
Maybe we can use containsOnly?

Reply:
why "containsOnly"? I think it should be "contains" from assertJ ListAssert

Reply:
well if we only use the contains operator the actual list could contain additional values. Using containsOnly we can verify that only those values are values of that list